doc/ref_model/chapters/chapter01.rst

```diff
@@ -0,0 +1,99 @@
+Introduction
+============
+
+Overview
+--------
+
+The Reference Model (RM) specifies a virtualisation technology agnostic (VM-based and container-based) cloud infrastructure abstraction and acts as a "catalogue" of the exposed infrastructure capabilities, resources, and interfaces required by the workloads. This document has been developed by the Linux Foundation Networking project `Anuket <../../common/chapter00.md>`__.
```

Comment:
```suggestion
The Reference Model (RM) specifies a virtualisation technology agnostic (VM-based and container-based) cloud infrastructure abstraction and acts as a "catalogue" of the exposed infrastructure capabilities, resources, and interfaces required by the workloads. This document has been developed by the Linux Foundation Networking project `Anuket <../../common/chapter00>`__.
```

Reply:
At the moment the correctness of this link relies on the relative link corrector extension what needs this to use ".md" extension. We will be able to correct only those links which are in the same pr as this as the correct linking needs the .rst files.